toolbox/db/db_get.m

```diff
@@ -257,7 +413,11 @@
                 else
                     condQuery.Id = iFiles(i);
                 end
-                sFiles(i) = sql_query(sqlConn, 'select', 'functionalfile', fields, condQuery);
+                result = sql_query(sqlConn, 'select', 'functionalfile', fields, condQuery);
+                if isempty(result)
+                    error('FunctionalFile entry was not found in db_get()')
```

Comment:
Similar comment as above, not a deal breaker but would be nice to get the missing functional file in the error message.

Reply:
Added in 1120e3143f6839ed14eb94df8af74e60fdd5cd23